apps/wallet/src/ui/app/components/transactions-card/TransactionsCard.module.scss

```diff
@@ -190,3 +190,19 @@
     color: colors.$gray-65;
     font-weight: 400;
 }
+
+.center-loading {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+}
+
+.tx-content {
```

Comment:
I am not sure we need this, there is this logic/styling already in the parent components